templates/config.php.j2

```diff
@@ -11,8 +28,8 @@ return [
         'database' => env('MYSQL_DATABASE', '{{ engelsystem_mysql_database }}'),
     ],
 
-    {% for k, v in engelsystem_config.items() %}
-    '{{ k }}' => {{ v }},
+    {% for key, value in engelsystem_config.items() %}
+    '{{ key }}' => {{ insert_var(value) }},
```

Comment:
Escape quotes etc. by using `to_json`. This also prevents PHP injection.

(I guess no valid key contains a character that needs escaping, so it would be an error. However, I think even then we shouldn't generate invalid PHP.)

```suggestion
    {{ key | to_json }} => {{ insert_var(value) }},
```